src/main/webapp/help/auth/agent-user-name-help.html

```diff
@@ -0,0 +1,3 @@
+<div>
+If you are using a cluster of Jenkins agents (e.g. using the swarm plugin) this is the user that is used for authenticating agents. This user will receive the overall read rights as well as rights to create, connect and configure agents.

```

Comment:
While true, this can be generalised to "inbound agents". I think that would more accurately follow the terminology from the Jenkins community.

For example:

```suggestion
If you are using inbound Jenkins agents, this is the user that is used for authenticating agents. This user will receive rights to create, connect and configure agents.
```